chromadb/test/property/test_add.py

```diff
@@ -179,22 +185,29 @@ def test_add_large(
         embedding_function=collection.embedding_function,
     )
     normalized_record_set = invariants.wrap_all(record_set)
-    initial_version = coll.get_model()["version"]
+    initial_version = cast(int, coll.get_model()["version"])
 
+    ids = []
     for batch in create_batches(
         api=client,
         ids=cast(List[str], record_set["ids"]),
         embeddings=cast(Embeddings, record_set["embeddings"]),
         metadatas=cast(Metadatas, record_set["metadatas"]),
         documents=cast(List[str], record_set["documents"]),
     ):
-        coll.add(*batch)
+        result = coll.add(*batch)
+        if result["ids"] is not None:
```

Comment:
result["ids"] should never be `None`; we can set the type on result as `result: IDs` or else cast, or else raise.